test/e2e/kubelet_stats.go

```diff
@@ -661,3 +695,36 @@ func (r *resourceMonitor) GetCPUSummary() nodesCPUSummary {
 	}
 	return result
 }
+
+func (r *resourceMonitor) GetMasterNodeCPUSummary(summaryPerNode nodesCPUSummary) nodesCPUSummary {
+	result := make(nodesCPUSummary)
+	var masterSummary containersCPUSummary
+	var nodesSummaries []containersCPUSummary
+	for node, summary := range summaryPerNode {
```

Comment:
Depending on name will hit us at some point... It's a real pain, as I'm not sure if we can have `map[api.Node]` or do we want to have `map[*api.Node]` in our codebase. In situation like this we'd need some mapping from name to `api.Node` structure which seems like an overkill. @bgrant0607 was opposing depending on names for distinguishing master components from other ones. I don't think there's a big difference between Pods and Nodes in this respect.